ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/RaoParameters.java

```diff
@@ -207,14 +207,15 @@ public void setMnecViolationCost(double mnecViolationCost) {
         Suppliers.memoize(() -> ExtensionProviders.createProvider(ConfigLoader.class, "rao-parameters"));
 
     /**
-     * Load parameters from platform default config.
+     * @return RaoParameters from platform default config.
      */
     public static RaoParameters load() {
         return load(PlatformConfig.defaultConfig());
     }
 
     /**
-     * Load parameters from a provided platform config.
+     * @param platformConfig PlatformConfig where the RaoParameters should be taken
```

Comment:
"platformConfig PlatformConfig where the RaoParameters should be read from" is clearer imo

Reply:
ok, thanks